corehq/apps/hqadmin/reports.py

```diff
@@ -185,6 +185,9 @@
         "chart_title": "Active Mobile Users (last 30 days)",
         "get_request_params": {
             "include_forms": True,
+            "additional_params_es": {
```

Comment:
just checking - this looks like it would include mobile users who received SMS - will it also include mobile users who sent SMS?